modules/apigee_edge_teams/src/Entity/ListBuilder/TeamListBuilder.php

```diff
@@ -74,7 +74,7 @@ protected function getDefaultOperations(EntityInterface $entity) {
       $members_url = $entity->toUrl('members');
       if ($members_url->access()) {
         $operations['members'] = [
-          'title' => $this->t('Members'),
+          'title' => $this->t('Team Members'),
```

Comment:
See 1st comment in apigee_edge_teams.links.action.yml.

Reply:
Updated PR.